.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,5 +2,6 @@
 
 **Description of changes:**
 
+**How you tested your changes:**
```

Comment:
Include additional questions like environment as well, so that they will specify linux or windows, k8s version and any specific detail

Reply:
Good idea. Will add those